node/core/av-store/src/lib.rs

```diff
@@ -564,7 +564,7 @@ where
 				log::trace!(
 					target: LOG_TARGET,
 					"Updating pruning record for finalized block {}",
-					record.candidate_hash,
+					record.block_number,
```

Comment:
I guess  this eases reading the output?

Reply:
It says "finalized_block" and the candidate hash was never and will never be a finalized block :P